### PR TITLE
Grammatical errors found fix #2945

### LINKED ISSUE #2945: Grammatical errors found in Ruby on Rails API
In https://auth0.com/docs/quickstart/backend/rails there are grammatical errors found:

1.
- Setup --> Set up
image

2.
- add comma
image

Auth0 ticket 3565

### PR DESCRIPTION
**Pull request by camimonse**
Setup --> Set up
add comma

### CODE REVIEW

**Comment by camimonse** [comment]
@chenkie the "fix #xx" like in this PR title?

**Comment by camimonse** [comment]
fix #2945

File: articles/server-apis/rails/authenticate.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ You can now make requests against your secure API by providing the Authorization
 
 ### 5. You're done!
 
-Now you have both your FrontEnd and Backend configured to use Auth0. Congrats, you're awesome!
+Now you have both, your FrontEnd and Backend configured to use Auth0. Congrats, you're awesome!
```
Comment (ntotten) [inline articles/server-apis/rails/authenticate.md]: I dont think this is right. The comma isnt needed. Also it should be "frontend" not "FrontEnd" and "backend", no need to capitalize.

**Comment by camimonse** [comment]
@chenkie ok! it is fixed now. What about the title, do we need for the automatic merge the "fix xx" to be like is her ? or how?

**Comment by mpaktiti** [comment]
@camimonse [Closing Issues via Pull Requests](https://github.com/blog/1506-closing-issues-via-pull-requests)

Looks good 👍

**Pull request merged**